valhalla/thor/timedep.h

```diff
@@ -1,8 +1,22 @@
-#ifndef VALHALLA_THOR_TIMEDEP_H_
-#define VALHALLA_THOR_TIMEDEP_H_
+#pragma once
```

Comment:
again, this class derived from astar and used a lot of its methods. so ive copied them over here